parachains/runtimes/collectives/collectives-polkadot/src/xcm_config.rs

```diff
@@ -140,7 +140,7 @@ impl xcm_executor::Config for XcmConfig {
 	// Collectives does not recognize a reserve location for any asset. Users must teleport DOT
 	// where allowed (e.g. with the Relay Chain).
 	type IsReserve = ();
-	type IsTeleporter = NativeAsset; // <- should be enough to allow teleportation of DOT
+	type IsTeleporter = ConcreteNativeAssetFrom<DotLocation>; // <- should be enough to allow teleportation of DOT
```

Comment:
Let me suggest adding one more `/` so it shows up in the doc.

```suggestion
	/// Only allow teleportation of DOT.
	type IsTeleporter = ConcreteNativeAssetFrom<DotLocation>; 
```